nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md

```diff
@@ -58,7 +58,7 @@ Several generic profiles are bundled with the pipeline which instruct the pipeli
 The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
 
 Note that multiple profiles can be loaded, for example: `-profile test,docker` - the order of arguments is important!
-They are loaded in sequence, so later profiles can overwrite earlier profiles.
+They are loaded in sequence, so parameters or settings in earlier profiles can overwrite later profiles.
```

Comment:
Is this right? That would mean that they are loaded in reverse sequence (from right to left) so that the last thing seen remains. I thought that what was already written was correct, but I'm not 100% sure.

Should be easy to check by doing test and an institute and seeing whether the config description says test or the institute name..

Reply:
Ok will I tested it, and it seems actually it doesn't make sense.

![image](https://user-images.githubusercontent.com/17950287/107182654-72399f80-69dd-11eb-8cc9-0aa205959462.png)

![image](https://user-images.githubusercontent.com/17950287/107182716-8d0c1400-69dd-11eb-82a5-66cb3a165206.png)

I just checked the nextflow docs again, and it appears (I guess I misunderstood something at some point) that the 'order' only seems the loading of the config files, not the specification of `-profiles`...

So maybe it doesnt matter anyway?

Reply:
The source of our confusion is a @nextflow-io bug! https://github.com/nextflow-io/nextflow/issues/1792 🐛 

So the docs as they stand _should_ be correct, but due to the bug the order or loading is not being proper respected currently. So I think that we should leave the docs unchanged on the assumption that this will go back to left-to-right again at some point soon.

Reply:
```suggestion
They are loaded in sequence, so later profiles can overwrite earlier profiles.
```